### ISSUE TITLE
`get_pixmap` - RuntimeError: Private data too large to pack into display list node

### ISSUE THREAD

**Issue by zcakzhuu** [pymupdf/PyMuPDF#2563]
_**Please provide all mandatory information!**_

## Describe the bug (mandatory)
`get_pixmap` worked for most of the pages in a document except a few pages. The problem only occurs when running over the pdf document downloaded straightaway from the website.

## To Reproduce (mandatory)
```
import fitz
import wget

url = "https://www.allianz.com/content/dam/onemarketing/azcom/Allianz_com/sustainability/documents/Allianz_Group_Sustainability_Report_2022-web.pdf"
document_path = wget.download(url, out=".")

doc = fitz.open(document_path)

page = doc[10]

pix = page.get_pixmap(dpi=200)
```

Error Message:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
[/var/tmp/ipykernel_29151/761847681.py](https://vscode-remote+ssh-002dremote-002bmengting-002dnlp-002dsandbox2.vscode-resource.vscode-cdn.net/var/tmp/ipykernel_29151/761847681.py) in ?()
----> 1 pix = page.get_pixmap(dpi=200)

[/opt/conda/envs/python38/lib/python3.8/site-packages/fitz/utils.py](https://vscode-remote+ssh-002dremote-002bmengting-002dnlp-002dsandbox2.vscode-resource.vscode-cdn.net/opt/conda/envs/python38/lib/python3.8/site-packages/fitz/utils.py) in ?(page, matrix, dpi, colorspace, clip, alpha, annots)
    891             colorspace = csRGB
    892     if colorspace.n not in (1, 3, 4):
    893         raise ValueError("unsupported colorspace")
    894 
--> 895     dl = page.get_displaylist(annots=annots)
    896     pix = dl.get_pixmap(matrix=matrix, colorspace=colorspace, alpha=alpha, clip=clip)
    897     dl = None
    898     if dpi:

[/opt/conda/envs/python38/lib/python3.8/site-packages/fitz/fitz.py](https://vscode-remote+ssh-002dremote-002bmengting-002dnlp-002dsandbox2.vscode-resource.vscode-cdn.net/opt/conda/envs/python38/lib/python3.8/site-packages/fitz/fitz.py) in ?(self, annots)
   6439 
   6440         CheckParent(self)
   6441 
   6442 
-> 6443         val = _fitz.Page_get_displaylist(self, annots)
   6444         val.thisown = True
   6445 
   6446         return val

RuntimeError: Private data too large to pack into display list node
```

## Your configuration (mandatory)

```
3.8.16 | packaged by conda-forge | (default, Feb  1 2023, 16:01:55) 
[GCC 11.3.0] 
 linux 
 
PyMuPDF 1.22.5: Python bindings for the MuPDF 1.22.2 library.
Version date: 2023-06-21 00:00:01.
Built for Python 3.8 on linux (64-bit).
```

 - Operating system, potentially version and bitness

> PRETTY_NAME="Debian GNU/Linux 10 (buster)"
> NAME="Debian GNU/Linux"
> VERSION_ID="10"
> VERSION="10 (buster)"
> VERSION_CODENAME=buster
> ID=debian
> HOME_URL="https://www.debian.org/"
> SUPPORT_URL="https://www.debian.org/support"
> BUG_REPORT_URL="https://bugs.debian.org/"
> BITNESS=64

 - Python version, bitness
 > Python 3.8.16
> Bitness 64
 - PyMuPDF version, installation method (**wheel** or **generated** from source).
> Version 1.22.5
> Installation Method: pip install using wheel

**Comment by watermeleon** [comment]
I am running into the same problem for another pdf and found your post. 
Please let me know if you found a workaround

**Comment by JorjMcKie** [comment]
Found the discrepancy between Linux and Windows. Will be fixed in next version.

**Comment by djoltes** [comment]
I just ran into this error on Windows while using PyMuPDFb-1.23.3 and an image resolution of 144dpi. Should this have been resolved, or do I need to dial down resolution temporarily?

**Comment by JorjMcKie** [comment]
@djoltes - please provide a reproducing file.

**Comment by djoltes** [comment]
> @djoltes - please provide a reproducing file.

Right now I can't -- only one page in one file has exhibited the behavior, and it's full of PII so I can't share it. I just wanted to make sure the bug had been resolved in this particular combination (Windows, Python 3.10, and PyMuPDF 1.23.3).

**Comment by JorjMcKie** [comment]
I understand. The bug has been fixed. It actually was limitation that restricted the dimensions of images displayed.
Whether or not your case falls under this can however not be decided based on the current formation.
So I suggest we close this for now.
Please reopen, whenever you can underpin your situation with reproducibale data.

**Comment by zhjiaooo** [comment]
I fixed it with version 1.20.2
